frontend/src/pages/article/components/author/styles.module.scss

```diff
@@ -0,0 +1,90 @@
+.container {
+  max-width: 466px;
+  background: #ebece7;
+  border: 1px solid var(--light-grey-border);
+  border-radius: 10px;
+}
+
+.authorWrpr {
+  display: flex;
+  align-items: center;
+  padding: 30px 41px 21px;
+  border-bottom: 2px solid rgba(172 186 215 / 20%);
```

Comment:
An here

Reply:
This is author-component from task/wr-53. I have merge those branch when worked on my task. I will merge those branch again with changes after review. I already linked your comments to wr-53.